tiledb/sm/query/deletes_and_updates/serialization.cc

```diff
@@ -51,8 +51,8 @@ void serialize_condition_impl(
     const auto field_name = node->get_field_name();
     const uint32_t field_name_length =
         static_cast<uint32_t>(field_name.length());
-    const auto& value = node->get_condition_value_view();
-    const storage_size_t value_length = value.size();
+    const auto& value = node->get_value_ptr();
```

Comment:
Could we add basic tests to cover IN/NOT IN in delete conditions?

Reply:
I've added a super basic test for this that triggers all the AST serialization code. I totes should have looked harder at that code when I was fixing syntax errors to realize I need this. Please take a look to see if there's any other major parts of the deletion API that I should add.